### PR TITLE
Refactored polish translation

### PR DESCRIPTION
**Pull request by kisielwrosole**
I refactored polish readme to match the English version. Also, I made changes proposed in the last pull request. I didn't know if closing that pull request was good, but this refactor would practically discard previous code. I would appreciate feedback :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -249,10 +249,7 @@ Controller controller = Get.put(Controller()); // Rather Controller controller =
 ```
 
 - Note: If you are using Get's State Manager, pay more attention to the bindings api, which will make easier to connect your view to your controller.
-
-Instead of instantiating your class within the class you are using, you are instantiating it within the Get instance, which will make it available throughout your App.
-So you can use your controller (or class Bloc) normally
-
+https://github.com/jonataslaw/ge
```
Comment (jonataslaw) [inline README.md]: Hi, I think this link was accidentally pasted here.

File: README.md [hunk 2]
```diff
@@ -457,7 +454,7 @@ GetMaterialApp(
   opaqueRoute: Get.isOpaqueRouteDefault,
   popGesture: Get.isPopGestureEnable,
   transitionDuration: Get.defaultDurationTransition,
-  defaultGlobalState: Get.defaultGlobalState,
+  defaultGlobalState: Get.defaultGlobalState,https://github.com/jonataslaw/ge
```
Comment (jonataslaw) [inline README.md]: And here too

File: README.md (hunk 1 above)
Comment (kisielwrosole) [reply README.md]: Yes indeed

**Comment by jonataslaw** [comment]
Wow, thanks for you contributation @kisielwrosole and @szymon-filipiak !
LGTM. 

@Nipodemos  do you have anything to add? As you take care of this sector of the lib, I will wait for you to see if everything is ok for you and then merge.

File: README.pl.md [hunk 3]
```diff
@@ -0,0 +1,527 @@
...
+
+I w UI, kiedy chcesz go zaktualizować przy modyfikacji zmiennej po prostu dodaj to:
+```dart
+Obx (() => Text (controller.name));
+```
+
+To wszystko. *Proste*, co nie?
+
+### Bardziej szczegółowo o menadżerze stanu
+**Zobacz bardziej szczegółowe wytłumaczenie menadz=żera sranu [tutaj](./docs/en_US/state_menagment.md). Znajdują się  tam przykłady jak o różnice między prostym menadżerem stanu oraz reaktywnym**
```
Comment (Nipodemos) [inline README.pl.md:179]:
```suggestion
**Zobacz bardziej szczegółowe wytłumaczenie menadz=żera sranu [tutaj](./docs/en_US/state_management.md). Znajdują się  tam przykłady jak o różnice między prostym menadżerem stanu oraz reaktywnym**
```

**Comment by Nipodemos** [comment]
i took the freedom of fix this tiny typo i found, i hope you don't mind @kisielwrosole 
@jonataslaw just reviewed and LGTM

**Pull request merged**